--- lapsus/dbus-qt4-qt3backport/Makefile.am~	2007-09-16 20:55:02.000000000 +0200
+++ lapsus/dbus-qt4-qt3backport/Makefile.am	2007-09-16 21:17:03.886292232 +0200
@@ -18,6 +18,8 @@
 	$(srcdir)/qdbusdatalist.cpp      \
 	$(srcdir)/qdbusobjectpath.cpp
 
+libdbus_1_qt3_la_LIBADD = -ldbus-1
+
 $(srcdir)/qdbusserver.cpp: qdbusserver.moc
 $(srcdir)/qdbusconnection.cpp: qdbusconnection.moc
 $(srcdir)/qdbusproxy.cpp: qdbusproxy.moc
